plugins/modules/proxysql_global_variables.py

```diff
@@ -156,9 +142,9 @@ def set_config(variable, value, cursor):
 def manage_config(variable, save_to_disk, load_to_runtime, cursor, state):
     if state:
         if save_to_disk:
-            save_config_to_disk(variable, cursor)
+            save_config_to_disk(cursor, "VARIABLES", variable)
```

Comment:
To be honest, I'm not sure to understand what the variable "variable" contains.
I think it is the name of the variable we are saving/loading.

But, why is this only used in this module? I don't know. Maybe there is more room for refactoring.

Reply:
@laurent-indermuehle yeah, feel free to refactor it more after this PR gets merged! Also there's a good reason to learn `pytest` for unit testing:)
you can find many examples in other collections, e.g. in community.mysql. Though be careful, I'm not sure my fake cursor class implementation is OK in general and adheres best practices;)